service/dps/storm/common/src/main/java/eu/europeana/cloud/service/dps/storm/StormTaskTuple.java

```diff
@@ -125,6 +119,10 @@ public Revision getRevisionToBeApplied() {
         return revisionToBeApplied;
     }
 
+    public void setRevisionToBeApplied(Revision revision) {
```

Comment:
Setter for the revision was removed in ECL-1071 as a result of my remark. The idea is not to give a possibility to change the revision after a task was submitted. All the necessary data should be passed in a constructor which also supports revision. I would remove it...